src/presentation/routers/user-router.ts

```diff
@@ -13,6 +15,31 @@ const UserRouter = (db: NoSQLWrapper) => {
         const resultDb = await db.CreateUser(user);
         response.status(HttpStateCodes.OK).json({response: resultDb});
     })
+    router.delete('/user/:id', async(request, response) => {
+        const {id}=request.params;
+        try {
+            const result = await db.DeleteUsers(id);
+            result
+            ? response.status(HttpStateCodes.OK).send('Successfully deleted user with id ${id}')
```

Comment:
Another very important thing isn't to hardcode the messages it is better to use a central place to set the message I explain very good that in this pr.
check it -> https://github.com/Ditmar/api_sem_g2/pull/2/files
![image](https://github.com/Ditmar/api_sem_g1/assets/317151/41230399-d3c6-43ce-ac1c-be3884aa98e6)